crates/biome_js_analyze/tests/specs/style/useNumericLiterals/invalid.js.snap

```diff
@@ -1,5 +1,6 @@
 ---
 source: crates/biome_js_analyze/tests/spec_tests.rs
+assertion_line: 83
```

Comment:
It seems you're running an old version of `insta`, because this line isn't needed anymore in new versions . Let's delete it :)
```suggestion
```